service/user/api/src/main/java/org/eclipse/kapua/service/user/UserType.java

```diff
@@ -0,0 +1,21 @@
+package org.eclipse.kapua.service.user;
```

Comment:
Forgot here as well. Adding them right away. Need to enforce them automatically...